pkg/machine/applehv/machine.go

```diff
@@ -79,32 +79,18 @@ type MacMachine struct {
 	GvProxySock machine.VMFile
 }
 
-func (m *MacMachine) Init(opts machine.InitOptions) (bool, error) {
-	var (
-		key          string
-		mounts       []machine.Mount
-		virtiofsMnts []machine.VirtIoFs
-	)
-	dataDir, err := machine.GetDataDir(machine.AppleHvVirt)
-	if err != nil {
-		return false, err
-	}
-	cfg, err := config.Default()
-	if err != nil {
-		return false, err
-	}
-
+func acquireVMImage(opts machine.InitOptions, m *MacMachine, dataDir string) error {
```

Comment:
Nit: please add as many comments as possible.  I think that the machine needs more comments and that is a nice moment to add some.

We can always add them later, so it's not a blocker.

Reply:
I definitely agree. I'll make a separate PR to add some documentation for the funcs

Reply:
added some documentation for the new functions that were added